omnibus/files/private-chef-ctl-commands/secrets.rb

```diff
@@ -42,16 +51,15 @@ def confirm_continue!(message)
   end
 end
 
-def set_secret(group, key, secret)
+def set(group, key, secret)
```

Comment:
`set_secret` did indeed clash with [this](https://github.com/chef/omnibus-ctl/blob/master/lib/omnibus-ctl.rb#L199) :D